draft-ietf-ppm-dap.md

```diff
@@ -1193,14 +1193,11 @@ it from the set of candidate reports.
 Next, for each report the Leader executes the following procedure:
 
 ~~~
-state = VDAF.ping_pong_start(vdaf_verify_key,
-                             True,
-                             agg_param,
-                             report_id,
-                             public_share,
-                             plaintext_input_share.payload)
-if state != Rejected():
-  (state, outbound) = VDAF.ping_pong_req(agg_param, state, None)
+(state, outbound) = Vdaf.ping_pong_leader_init(vdaf_verify_key,
+                                               agg_param,
+                                               report_id,
+                                               public_share,
+                                               plaintext_input_share.payload)
```

Comment:
I'm pretty sure the `make` winds if "art" (lines terminated by "~~~") is longer than 70 columns.

Reply:
We end up with this:

```
   Next, for each report the Leader executes the following procedure:

(state, outbound) = Vdaf.ping_pong_leader_init(vdaf_verify_key,
                                               agg_param,
                                               report_id,
                                               public_share,
                                               plaintext_input_share.payload)

   where:
```

which didn't make the renderer bug out, but it is weird how the centered text goes past the left margin. I re-indented and now we get:

```
   Next, for each report the Leader executes the following procedure:

   (state, outbound) = Vdaf.ping_pong_leader_init(
       vdaf_verify_key,
       agg_param,
       report_id,
       public_share,
       plaintext_input_share.payload)

   where:
```

which I think is better.